UPGRADING.md

```diff
@@ -3,6 +3,14 @@
 This guide provides steps to be followed when you upgrade your applications to
 a newer version of Tendermint Core.
 
+## Unreleased
+
+<Overview>
+
+### Protobuf Changes
+
+When upgrading to version <version #> you will have to fetch the third_party directory along with the updated proto files.
```

Comment:
```suggestion
When upgrading to version <version #> you will have to fetch the `third_party` directory along with the updated proto files.
```

Reply:
also, when you say "fetch," you literally mean getting them from the same repo as the proto files, right? just want to make sure we don't need to include more instructions.

Reply:
yes that is correct. I can link to rust-abci pr I have to do this for more context if you think it would help.